lz4.go

```diff
@@ -23,7 +24,7 @@ func (lz Lz4) Match(filename string, stream io.Reader) (MatchResult, error) {
 	var mr MatchResult
 
 	// match filename
-	if strings.Contains(strings.ToLower(filename), lz.Name()) {
+	if filepath.Ext(strings.ToLower(filename)) == lz.Name() {
```

Comment:
Why was this one changed?

Reply:
I thought it would be a conflict, but it was a misunderstanding.